src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -63,154 +65,173 @@ The following table provides some examples of our use of code format.
       <td>
         Method names
       </td>
-
+      <td>
+        Using code formatting for method names is standard practice.
+      </td>
       <td>
         To initialize the APM agent, use `startAgent()`.
       </td>
     </tr>
 
     <tr>
       <td>
-        Non-clickable URLs
+        URLs you don't intend to be clicked
+      </td>
+      <td>
+        Using code formatting hints to the user that they may need to customize or tweak the URL to ensure it works for them.
       </td>
-
       <td>
         In your web browser, navigate to the minion **Overview** page at `http://MINION_IP_ADDRESS`.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Command line utility names
+      </td>
+      <td>
+        If you think a reader or translator might confuse a command with a general English word, use code. 
 
-        The `<var>` formatting automatically applies color coding and italics in this situation.
+          * `curl`: Sends http requests via a terminal session–not to be confused with the curls you do with weights at the gym.
+          * `cat`: Lists the first ten lines of a file–not a feline pet who ignores you.
+          * `date`: Displays the day, year, and time–not an outing that couples take.
+          * `tail`: Displays the last ten lines of a file–not the appendage on various mammals.
+          * `which`: Show the location of a program executable–not the pronoun.
+
+      </td>
+      <td>
+        To install the utility, use `apt`.
       </td>
     </tr>
+
   </tbody>
 </table>
 
+## Inline code formatting [#inline-code]
+
+To surround small blocks of code or data (single words or lines), surround the text with backtick `` ` `` characters. 
 
-These sections break down when to use specific types of code formatting.
+## Multi-line code blocks [#code-block]
+  
+To format one or more lines of code, insert three backticks ( `` ``` `` ) above and below your code. This essentially acts like the HTML `<pre>` tag.
 
 <CollapserGroup>
   <Collapser
-    id="inline-code"
-    title={<>For inline code or data (<InlineCode>&lt;code></InlineCode>)</>}
+    id="synax-highlight-example"
+    title="Syntax highlighting example"
   >
-    To surround small blocks of code or data (single words or lines), mark as code:
-
-    * **Source**: Use `<code>` tags.
-    * **Markdown**: Surround the text with backtick `` ` `` characters.
+  To add syntax highlighting to a code block, insert the language name on the same line as the first three backtacks. Here's an example that formats the handlebars template language:
+
+  ![Screenshot showing how to format a handlebars code block](./images/code-examples.png "Screenshot showing how to format a handlebars code block")
```

Comment:
@austin-schaefer, the image text is larger than the surrounding text so if you have time, you might make this image smaller.

Reply:
Hah! I kinda liked how giant it was so it plays at a variety of column widths.

Reply:
I'll keep it as is but feel free to downsize it if it's annoying you.